src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/DraftApprenticeship/ViewDraftApprenticeshipViewModelMapper.cs

```diff
@@ -44,7 +44,9 @@ public async Task<IDraftApprenticeshipViewModel> Map(ViewDraftApprenticeshipRequ
                 StartDate = draftApprenticeship.StartDate,
                 EndDate = draftApprenticeship.EndDate,
                 Reference = draftApprenticeship.Reference,
-                LegalEntityName = source.Cohort.LegalEntityName
+                LegalEntityName = source.Cohort.LegalEntityName,
+                EmploymentEndDate = draftApprenticeship.EmploymentEndDate,
+                EmploymentPrice = draftApprenticeship.EmploymentPrice
```

Comment:
We should always include the trailing comma.  This makes future PR differences better by removing the irrelevant change to this line merely to add a comma.